apps/barista/src/components/live-example/live-example.ts

```diff
@@ -24,19 +24,24 @@ import {
   ViewContainerRef,
   ComponentRef,
   OnDestroy,
+  ViewEncapsulation,
 } from '@angular/core';
+import { coerceBooleanProperty } from '@angular/cdk/coercion';
 import { EXAMPLES_MAP } from '@dynatrace/barista-components/examples';
+import * as hljs from 'highlight.js';
```

Comment:
Please don't import all highlight this will increase the bundle size extremely only import the typescript, html and css packages. Please check bundle size after your changes